### PR TITLE
corrected typos in demography.md

### PR DESCRIPTION
**Pull request by cdquinto**

### CODE REVIEW

File: docs/demography.md [hunk 1]
```diff
@@ -301,7 +301,7 @@ demography
 
 :::{warning}
 We use the {meth}`.Demography.isolated_model` function here as a
-convenient means of creating example demographies. However, it's
+convenient mean of creating example demographies. However, it's
```
Comment (jeromekelleher) [inline docs/demography.md:304]: I think "means of" was technically correct. How about we change to "way to", as being a bit clearer?

**Review by jeromekelleher — COMMENTED** [review]
Thanks @cdquinto! One clarification above.

File: docs/demography.md (hunk 1 above)
Comment (cdquinto) [reply docs/demography.md]: got it!

**Review by jeromekelleher — APPROVED** [review]
Thanks @cdquinto!

**Comment by jeromekelleher** [comment]
@mergifyio rebase

**Pull request merged**